packages/explat-client/src/internal/test-common.ts

```diff
@@ -0,0 +1,14 @@
+export const validExperimentAssignmentA = {
```

Comment:
Should this file also be under `internal/test`?

Reply:
I tried to get it to work like that but the conventions don't allow it - they don't allow exports.